crates/codegen/runtime/cargo/wasm/src/runtime/wrappers/ast/generated/selectors.rs

```diff
@@ -83,15 +83,15 @@ impl Helper {
             let Edge { label, node } = self.node.children.get(self.index)?;
```

Comment:
now that they are no longer optional, I suggest returning `Option<Edge>` directly from this function..